test/table-constructor.ts

```diff
@@ -7,7 +7,8 @@ import {DefaultSchema} from './schemas'
 // jest.setTimeout(7200 * 1000)
 
 let TableName = 'TableConstructorTestTable'
-let table: Table = null
+// Schema need to be passed because its not in the constructor. Using any because it gets modified
+let table: Table<any>
```

Comment:
@mobsense here I had to pass any in the schema cause it gets modified

Reply:
Ok am I now using any in the type if its not provided here. Again this will only happen if the table is created without a schema